apps/institution-service/src/controllers/institutionController.test.ts

```diff
@@ -707,6 +714,106 @@ describe("institutionController", () => {
 
       expect(inactiveInstitutionFound).toBeFalsy();
     });
+
+    it("calls getSortOption with default sort order if sortBy is not passed in to request", async () => {
```

Comment:
These tests are really mocky, so they're testing implementation details instead of actual behavior

https://kentcdodds.com/blog/testing-implementation-details